App/src/main/java/com/gluonhq/chat/service/CloudlinkService.java

```diff
@@ -56,8 +59,8 @@ public void postConstruct() {
         getRemoteUsers();
     }
 
-    public GluonObservableList<ChatMessage> getMessages() {
-        return getMessages(null);
+    public ObservableList<ChatMessage> getMessages() {
```

Comment:
Add some basic JavaDoc here (i.e. to explain the difference between getMessages() and getMessages(Channel).